[2025-04-04T20:56:17.110+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: proyecto_bigdata.modelo_ml scheduled__2025-04-03T00:00:00+00:00 [queued]>
[2025-04-04T20:56:17.124+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: proyecto_bigdata.modelo_ml scheduled__2025-04-03T00:00:00+00:00 [queued]>
[2025-04-04T20:56:17.125+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-04T20:56:17.143+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): modelo_ml> on 2025-04-03 00:00:00+00:00
[2025-04-04T20:56:17.148+0000] {standard_task_runner.py:60} INFO - Started process 272 to run task
[2025-04-04T20:56:17.152+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'proyecto_bigdata', 'modelo_ml', 'scheduled__2025-04-03T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/proyecto_bigdata_dag.py', '--cfg-path', '/tmp/tmp8d512tyc']
[2025-04-04T20:56:17.155+0000] {standard_task_runner.py:88} INFO - Job 7: Subtask modelo_ml
[2025-04-04T20:56:17.229+0000] {task_command.py:423} INFO - Running <TaskInstance: proyecto_bigdata.modelo_ml scheduled__2025-04-03T00:00:00+00:00 [running]> on host 8cda24561222
[2025-04-04T20:56:17.334+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='proyecto_bigdata' AIRFLOW_CTX_TASK_ID='modelo_ml' AIRFLOW_CTX_EXECUTION_DATE='2025-04-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-03T00:00:00+00:00'
[2025-04-04T20:56:17.336+0000] {logging_mixin.py:188} INFO - ðŸ¤– Entrenando modelo de machine learning
[2025-04-04T20:56:17.337+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-04T20:56:17.352+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=proyecto_bigdata, task_id=modelo_ml, execution_date=20250403T000000, start_date=20250404T205617, end_date=20250404T205617
[2025-04-04T20:56:17.406+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-04T20:56:17.462+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
